packages/lib-classifier/src/store/Project/Project.js

```diff
@@ -17,7 +19,11 @@ const Project = types
         [singleActiveWorkflow] = self.links['active_workflows']
       }
       return singleActiveWorkflow
-    }
+    },
+
+    get display_name() {
+      return self.strings.get('display_name')
+    },
```

Comment:
Why is `display_name` changed to a `get` function instead of the model property?

Reply:
The model property doesn't exist in the incoming snapshot, leading to the error in #5073. Since translations were added, the display name is stored in `project.strings.display_name`.